pkg/tfgen/convert_cli.go

```diff
@@ -645,14 +649,14 @@ func (cc *cliConverter) singleExampleFromHCLToPCL(path, hclCode string) (transla
 func (cc *cliConverter) singleExampleFromPCLToLanguage(example translatedExample, lang string) (string, error) {
 	var err error
 
-	if example.PCL == "" {
-		return "", nil
-	}
 	source, diags, _ := cc.convertPCL(example.PCL, lang)
 	diags = cc.postProcessDiagnostics(diags.Extend(example.Diagnostics))
 	if diags.HasErrors() {
-		source = "Example currently unavailable in this language\n"
-		err = fmt.Errorf("failed to convert an example: %s", diags.Error())
+		err = fmt.Errorf("conversion errors: %s", diags.Error())
+	}
+
+	if source == "" {
+		source = exampleUnavailable
 	}
```

Comment:
I don't understand why this change makes sense:

If there are no errors (`!diag.HasErrors()`), but `source == ""`, should we really set `source = exampleUnavailable`.

Reply:
Two reasons:

1. It is possible for `diag` to have no errors (for example, when the input is empty), which results in `source` being empty. We can return an empty string, or declare that we currently have no example.

1. It is also possible for `diag` to _have_ errors but the conversion being successful regardless. 

Therefore it makes more sense to check for "what are we looking to display to the user" and if it comes up empty, say that.